.github/workflows/skyux-ci.yml

```diff
@@ -5,9 +5,11 @@ on:
   pull_request:
     branches:
       - master
+      - 5.0.0-next
   push:
```

Comment:
Does this need to go into master? Probably harmless - but I figured I would ask.